docs/getting_started.md

```diff
@@ -104,7 +104,7 @@ cd textual/examples/
 python code_browser.py ../
 ```
 
-
+Also, the tree of directories under `docs/examples/` contains hundreds of examples for all parts of Textual. 
```

Comment:
These are examples for the docs. They don't make much sense out of context. i.e. I wouldn't want to give the impression that the examples are the recommended approach for apps.

Reply:
In practice, a large number of programmers read the code over the docs.   That is, grep the entire set of examples for the usage of some function rather than start with the docs of the widget which, somewhere in the page, may mention the function.   

Let me see if better wording would clarify this.